web-animations-1/Overview.bs

```diff
@@ -5721,11 +5726,14 @@ animation.play();</pre>
 
     </div>
 
-:   <dfn method for=Animatable lt="getAnimations()">
-    sequence&lt;Animation&gt; getAnimations()</dfn>
+:   <dfn method for=Animatable lt="getAnimations(options)">
+    sequence&lt;Animation&gt; getAnimations(options)</dfn>
 ::  Returns the set of [=relevant=] {{Animation}} objects
     that contain at least one <a>animation effect</a> whose
-    <a>target element</a> is this object.
+    <a>target element</a> is this object, or,
+    if an {{Animatable/getAnimations(options)/options}} parameter
+    is passed with {{GetAnimationsOptions/subtree}} set to true,
+    an [=inclusive descendant=] of this object.
```

Comment:
Do we need to specify anything regarding shadow DOM, or does inclusive descendant automatically exclude them?

Reply:
As I understand it, "inclusive descendant" automatically excludes them. To include them you need to use the [shadow including inclusive descendant](https://dom.spec.whatwg.org/#concept-shadow-including-inclusive-descendant) concept.

Reply:
Ack, thanks.